src/Compilers/CSharp/Test/Syntax/Parsing/IgnoredDirectiveParsingTests.cs

```diff
@@ -171,26 +171,22 @@ public void ShebangNotFirst(bool script, bool featureFlag)
 
         VerifyTrivia();
         UsingTree(source, options,
-            // (1,2): error CS1024: Preprocessor directive expected
+            // (1,2): error CS1040: Preprocessor directives must appear as the first non-whitespace character on a line
```

Comment:
It looks like the directive *is* the first non-whitespace character on the line in this case.

Reply:
Agree, that this is somewhat confusing, though I'm not totally sure its worth fixing.